sources/packages/web/src/services/InstitutionService.ts

```diff
@@ -170,14 +171,16 @@ export class InstitutionService {
     return ApiClient.Institution.getUserTypeAndRoles();
   }
 
-  public async createUser(data: InstitutionUser): Promise<void> {
+  public async prepareUserPayload(isNew:boolean, data:InstitutionUserAuthDetails) {
```

Comment:
Is this method being used outside the service? If not, please consider changing to private.
I saw a similar one on Vue page, is it possible to move that logic here?